src/puya/awst/nodes.py

```diff
@@ -252,6 +319,14 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 @attrs.frozen
 class AssertExpression(Expression):
+    """
+    Asserts the `condition` is true and errors with `error_message` if it is not. If no condition
```

Comment:
Probably needs to mention that the message is only a comment in the TEAL, so it can't be dynamic in any way, and relies on the caller to extract it from the source...